src/commons/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import {ComponentMeta, ComponentStory} from '@storybook/react';
+
+import Button, {ButtonSize, ButtonStyle} from './Button';
+
+export default {
+    title: 'Example/Button',
+    component: Button,
+  } as ComponentMeta<typeof Button>;
+
+const Template: ComponentStory<typeof Button> = (args) => <Button
+    {...args} />;
+
+export const General = Template.bind({});
+// More on args: https://storybook.js.org/docs/react/writing-stories/args
+General.args = {
+  buttonSize: ButtonSize.General,
+  children: 'Button',
+  isLoading: false,
+  isDisabled: false,
+};
```

Comment:
If you create a Default story here, you don't need to provide args. Storybook will parse props and will show you controls.